.github/workflows/ci.yml

```diff
@@ -46,6 +48,10 @@ jobs:
     - name: Checkout code
       uses: actions/checkout@v2
 
+    # So GoReleaser can generate the changelog properly
+    - name: Unshallow
```

Comment:
Could use a better name here, at a glance it's unclear what this does when looking at the CI output

Reply:
Also maybe only run this if it's a tagged run?